src/main/java/racingcar/dao/DbGameDao.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.dao;
+
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import java.util.List;
+import java.util.Map;
+import javax.sql.DataSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class DbGameDao implements GameDao{
+    private final JdbcTemplate jdbcTemplate;
+
+    public DbGameDao(DataSource dataSource) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+    }
```

Comment:
인자로 JdbcTemplate를 바로 주입해줄 수도 있는데 DataSrouce를 사용한 이유가 있을까?

```suggestion
    public DbGameDao(JdbcTemplate jdbcTemplate) {
        this.jdbcTemplate = jdbcTemplate;
    }
```